testing.py

```diff
@@ -0,0 +1,5 @@
+import requests
+
```

Comment:
Nice, this should work in most cases. 
However, what is going to happen if a site does not accept your request or a page to which link leads simply does not exist? Would there be a way to prevent your program from crashing? 
It may turn out useful to consider such cases.